sunpy/timeseries/metadata.py

```diff
@@ -20,6 +20,14 @@ class TimeSeriesMetaData:
     multiple `sunpy.timeseries.TimeSeries` metadata to be concatenated in an
     organized fashion.
 
+    Possible signatures::
+
+        TimeSeriesMetaData(meta=dict, timerange=TimeRange, colnames=list)
+        TimeSeriesMetaData(meta=tuple)
```

Comment:
Am I missing something here, but it seems that` TimeSeriesMeta` needs a timerange no? If only a meta is given and no timerange the `TimeSeriesMeta` that is returned is empty - i.e. the meta is not saved within the object.

Reply:
You can give the `meta` kwarg a tuple of `(TimeRange, list, dict)` here: https://github.com/sunpy/sunpy/pull/3709/files#diff-5c1988a7acaca3d3c5035b0f0808a8c3R95 I think.